tests/pact/tc.repl

```diff
@@ -335,3 +335,36 @@
  "rtc object lists"
  1
  (rtc-object-list (let ((a {'a:1}) (b {'a: 2})) [a b])))
+
+;; A module with incorrect annotations for a function return type.
+;; Runtime typechecking for function return types was added in
+;; pact-4.7.
+(module tc-test-function-return-types gov
+  (defcap gov () true)
+  (defun f:string (a: string) (+ 1 2))
+  (defun g:integer (a: integer) a))
+
+;; At pact version 4.6 and earlier, the functions with incompatible type
+;; annotations should compose.
+(env-exec-config ["DisablePact45"])
```

Comment:
Doesn't this statement cause you to effectively use Pact 4.4?

Reply:
Once @jwiegley's other comment is addressed this should follow idiomatic procedures. Here you would "DisablePact47"...

Reply:
It seems like I have the wrong idea about the meanings of these flags. Will straighten it out.

Reply:
Thanks, gochya - updated.